snap/snapcraft.yaml

```diff
@@ -26,15 +26,15 @@ parts:
     source: .
     source-type: git
     plugin: cmake
-    build-packages: [build-essential, libboost-all-dev, libcvc4-dev]
+    build-packages: [build-essential, libboost-all-dev]
     stage-packages: [libicu60]
     override-build: |
       if git describe --exact-match --tags 2> /dev/null
       then
         echo -n > ../src/prerelease.txt
       fi
       snapcraftctl build
-    after: [z3]
+    after: [z3, cvc4]
   z3:
     source: https://github.com/Z3Prover/z3.git
     source-tag: z3-4.8.4
```

Comment:
By the way there's ``4.8.5`` now (and there will probably soon be ``4.8.6``) and it'd be very good to keep this up to date, because the newer Z3 versions noticeably increase the solving power of ``SMTChecker``...

Reply:
I can do that update in a separate branch.
It should be just updating the tag and everything should just work :tm: But I will give it a try.